test/policy-framework/start

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+import drenv
+
+HUB_NAMESPACE = "open-cluster-management"
+CLUSTER_NAMESPACE = "open-cluster-management-agent-addon"
+
+def deploy_hub(hub):
+    drenv.log_progress(
+        f"Installing governance policy framework in cluster {hub}",
+    )
+    drenv.run(
+        "clusteradm", "install", "hub-addon",
+        "--names", "governance-policy-framework",
+        "--context", hub,
+    )
```

Comment:
logging and installing with clusteradm is repeated here, and will be repeated in 
many scripts. I think we need a clusteradm wrapper module to make this nicer to use.

Something like:

    clusteadm.install(command, names, cluster)
    clusteradm.addon(command, name, clusters)